examples/flows/standard/conditional-flow-for-if-else/llm_result.py

```diff
@@ -0,0 +1,10 @@
+from promptflow import tool
+
+
+@tool
+def llm_result(question: str) -> str:
+    # You can use an LLM node to replace this tool.
```

Comment:
Do we intentionally to not really call LLM here?

Reply:
Yes. Because the goal of this sample flow is to show how to use `activate config`, not how to call LLM. If we call the real LLM here, customers must have an aaoai connection. We don't want this flow to have too many dependencies and become complicated.